packages/cfpb-atomic-component/src/utilities/atomic-helpers.js

```diff
@@ -115,7 +116,7 @@ function instantiateAll(selector, Constructor, scope) {
     element = elements[i];
     if (contains(element, INIT_FLAG) === false) {
       inst = new Constructor(element);
-      inst.init();
+      inst.init(config);
```

Comment:
Thank you for this! This got me thinking about the architectural approach of configuring components at initialization.

On the one hand it's nice to have a call to `instantiateAll` always return the same thing, without any additional arbitrary config. However, we do need a way to specify different initial states of a component in some cases, such as the config you have in this PR. 

I looked at how we do this with Expandables. In Expandables, there is an option where an Expandable can be open on page load. For this, we add a CSS class [`o-expandable__onload-open`](https://github.com/cfpb/design-system/blob/fb976fcacecc7e5439b822e022be96befe1cbd6a/packages/cfpb-expandables/src/Expandable.js#L49). This class itself adds no styles directly (using JS to set additional CSS that sets the appearance). Per BEM naming, the `o-expandable` sets a certain appearance, and `o-expandable__onload-open` is a modifier on that base appearance. 

The trade offs of this are:

- Since the flag for this configuration appears in the markup, the site can be searched using the crawler for `o-expandable__onload-open` to find all expandables that are open on load. 
- This is possibly an abuse of BEM, since the modifier, `o-expandable_onload-open`, does not actually change the appearance directly itself.

Now, in regard to the multiselect, I could see the max selections being an attribute on the HTML, in the same way the native `<select>` element has a `size` attribute, for example. However, since the native HTML has a limited set of attributes, we would either need to add a custom `data-max-selections` attribute, or take a class-based approach, as we do with expandables. However, neither of these approaches are elegant. I think in the future, the multiselect will probably be a web component and we would set an attribute on the custom element.

So, in conclusion, I think this config approach on `instantiateAll` is fine, but two things should happen in the future through someone's effort:

- Expandables should align better with this same pattern.
- The configuration options should likely be set through some sort of attribute that appears in the markup.